src/main/resources/db/migration/common/V1_204__alter_device_register_table.sql

```diff
@@ -0,0 +1,7 @@
+DROP SEQUENCE IF EXISTS der_seq;
+
+ALTER TABLE device_register DROP CONSTRAINT device_register_pk;
+
+ALTER TABLE device_register RENAME TO node_register;
+
+CREATE SEQUENCE nod_seq START WITH 10000;
```

Comment:
Sorry - I thought I left a comment here, wondering why the sequence was starting at 10000?

Reply:
It is to catch any existing ids. It was increased to 10,000.